util/dashboard/fp_011.py

```
@@ -19,13 +22,14 @@
 ## ```
 ##
 ## ### Implementation
+##
 ## The registry data entry is validated with JSON schema using the [contact schema](https://raw.githubusercontent.com/OBOFoundry/OBOFoundry.github.io/master/util/schema/contact.json). The contact schema ensures that a contact entry is present and that the entry has a name and email address.
 
-import jsonschema
-
 import dash_utils
```

Ruff

`dash_utils` imported but unused

Remove unused import: `dash_utils`

(F401)